tailwind.config.js

```diff
@@ -50,6 +50,8 @@ module.exports = {
       sm: '0px 0px 1px rgba(40, 51, 65, 0.2), 0px 1px 4px rgba(40, 51, 65, 0.1)',
       md: '0px 3px 12px rgba(40, 51, 65, 0.1), 0px 0px 1px rgba(40, 51, 65, 0.2)',
       lg: '0px 4px 16px rgba(40, 51, 65, 0.1), 0px 0px 1px rgba(40, 51, 65, 0.2)',
+      light: '0px 0px 0px 4px #D6E4F7',
+      dark: '0px 0px 0px 4px #41546C',
```

Comment:
I wonder if we could improve the naming here. Looking at the options in the tailwind config doesn't really immediately look like "input focus state shadow". Not quite sure what this could be... Maybe just `light-focus` and `dark-focus`?

Reply:
Updated based on your suggestion